src/molecules/Avatar/Avatar.jsx

```diff
@@ -1,14 +1,49 @@
-import React from 'react';
-import { AvatarImg } from './styled';
-import { ButtonAtom } from '../../atoms';
-import Icon from '../../assets/img/profile_icon.png';
+import React, { useState, useContext } from 'react';
+import { Link, useHistory } from 'react-router-dom';
+import { AvatarImg, RelativePos } from './styled';
+import { ButtonAtom, MenuAtom, TextAtom } from '../../atoms';
+import { GlobalContext } from '../../states/GlobalContext';
+import PropTypes from 'prop-types';
 
 function Avatar({ className }) {
+  let [hide, setHide] = useState(true);
+  const [state, dispatch] = useContext(GlobalContext);
+  const history = useHistory();
+  const defaultImg = require('../../assets/img/profile_icon.png');
```

Comment:
This require should not be placed inside the component. Import the `defaultImg` at the top with the rest of the imports.